docs/en/ingest-management/release-notes/release-notes-8.11.asciidoc

```diff
@@ -57,6 +132,24 @@ Review important information about {fleet-server} and {agent} for the 8.11.2 rel
 
 IMPORTANT: The memory leak <<known-issue-115-8.11.1,known issue>> that affects Windows users running {agent} is resolved in this release. If you're currently on {agent} version 8.11.0 or 8.11.1, we strongly recommend upgrading to 8.11.2 or a later release to avoid the issue. If you're on an earlier version, avoid upgrading to version 8.11.0 or 8.11.1 and update directly to version 8.11.2 or later.
 
+[discrete]
+[[known-issues-8.11.2]]
+=== Known issues
+
+[[known-issue-169826-8.11.2]]
+.Triggering {agent} upgrades from {kib} {fleet} UI in an air-gapped environment will fail
+[%collapsible]
+====
+
+*Details*
+
+When attempting to upgrade an {agent}, {kib} tries to access (https://www.elastic.co/api/product_versions)[available versions]. 
```

Comment:
```suggestion
When attempting to upgrade an {agent}, {kib} tries to access https://www.elastic.co/api/product_versions.  
```

Reply:
The results of this link won't be particularly helpful to a user, so I'm proposing that we reverse my comment.